parser/tests/basic.rs

```diff
@@ -188,6 +188,8 @@ y
 #[test]
 fn expression() {
     let _ = ::env_logger::init();
+    let e = parse("2 * 3 + 4");
+    assert_eq!(e, Ok(binop(binop(int(2), "*", int(3)), "+", int(4))));
```

Comment:
Oops, were testing the next test case in this function so I removed this and forgot to add it back. :sweat: